com.unity.mobile.notifications/Runtime/Android/Plugins/com/unity/androidnotifications/UnityNotificationManager.java

```diff
@@ -53,6 +56,7 @@
     protected static final String KEY_NOTIFICATION = "unityNotification";
     protected static final String KEY_SMALL_ICON = "smallIcon";
     protected static final String KEY_CHANNEL_ID = "channelID";
+    protected static final String SHOW_IN_FOREGROUND = "showInForeground";
```

Comment:
Please, namespace the value. Other values are mostly legacy. Going forward, it's a good idea to name values liek "com.unity.xxx".

Reply:
Yes makes sense 👍🏻 Updated the value to "com.unity.showInForeground"